spec/csv_row_model/import/represents_spec.rb

```diff
@@ -0,0 +1,207 @@
+require 'spec_helper'
+
+describe CsvRowModel::Import::Represents do
+  let(:klass) { Class.new(BasicImportModel) { def self.name; "RepresentsTestModel" end } }
+  let(:instance) { klass.new }
+
+  describe "instance" do
+    describe "#valid?" do
+      subject { instance.valid? }
+
+      it "calls #filter_errors" do
+        expect(instance).to receive(:filter_errors)
+        subject
+      end
+    end
+
+    describe "#filter_errors" do
+      subject { instance.send :filter_errors }
+      before do
+        klass.send(:represents_one, :test_model, dependencies: %i[string1]) { "test" }
```

Comment:
`string1` is a column in `BasicImportModel` (`klass` subclasses this), yours is easier to read. i'm not sure if it's worth making a column just for the test :p